api-tests/features/p2-m2b.feature

```diff
@@ -0,0 +1,34 @@
+@Build
+Feature: M2B No Photo Id Journey
+
+  Scenario: M2B Journey
+    Given I start a new 'medium-confidence' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'end' event
+    Then I get a 'page-ipv-identity-postoffice-start' page response
+    When I submit an 'end' event with feature set 'm2bBetaExperianKbv'
```

Comment:
I would probably set a featureset in a separate step that applies to all requests:
```
  Scenario: M2B Journey
    Given I start a new 'medium-confidence' journey
    And I have the 'm2bBetaExperianKbv' featureset enabled
    Then I get a 'page-ipv-identity-document-start' page response
    ...
```